src/components/AddItems.js

```diff
@@ -1,8 +1,9 @@
-import React, { useState, useEffect } from 'react';
+import React, { useState } from 'react';
 import { db } from '../lib/firebase';
 import './AddItems.css';
 
-const userToken = 'token';
+//token to represent localStorage value
+const userToken = localStorage.getItem('tcl13-token');
```

Comment:
I tried adding an item after I created a list and the token showed up as null in the db. I believe all you need to do is move the userToken variable inside the AddItems component and you'll be all set.

![Screen Shot 2020-10-18 at 1 11 43 PM](https://user-images.githubusercontent.com/26862959/96378830-a0038b80-1143-11eb-80b0-3b426aa295a5.png)

Reply:
@AliLynne I just pushed up a new version with the variable inside the component. It was working for me over here. Let me know if you can get it to work!